content/sensu-core/1.8/migration.md

```diff
@@ -0,0 +1,380 @@
+See the [installation guide][54] to install and configure sensuctl.
+
+### 4. Set up Sensu users
+
+Role-based access control (RBAC) is a built-in feature of Sensu Go.
+RBAC allows management and access of users and resources based on namespaces, groups, roles, and bindings.
+To learn more about setting up RBAC in Sensu Go, see the [RBAC reference][38] and the [guide to creating a read-only user][39].
+
+In addition to built-in RBAC, Sensu includes [license-activated][40] support for authentication using Microsoft Active Directory and standards-compliant Lightweight Directory Access Protocol tools like OpenLDAP.
+
+### 5. Install agents
+
+“Clients” are now represented within Sensu Go as abstract “entities” that can describe a wider range of system components (network gear, web server, cloud resource, etc.)
+Entities include “agent entities” (entities running a Sensu agent) and familiar “proxy entities”.
+
+The Sensu agent is available for Ubuntu/Debian, RHEL/CentOS, Windows, and Docker.
+See the [installation guide][53] to install, configure, and start Sensu agents.
+
+Make sure to include the localhost subscription by uncommenting the subscription section and adding `localhost` to the list of subscriptions, or by using the `--subscriptions localhost` flag.
+
+{{< highlight yml >}}
+##
+# agent configuration
+##
+...
+subscriptions:
+ - "localhost"
+...
+{{< /highlight >}}
+
+Now you should have Sensu installed and functional. The next step is to translate your Sensu 1.x configs to Sensu Go.
+
+## Translate your configuration
+
+The [Sensu translator][18] is a command-line tool to help you transfer your Sensu Core 1.x checks, handlers, and mutators to Sensu Go.
+
+### 1. Run the translator
+
+Before we start, output your Sensu 1.x configuration to use as a reference.
+
+{{< highlight shell >}}
+# Requires curl and jq
+curl -s http://127.0.0.1:4567/settings | jq . > sensu_config_original.json
+{{< /highlight >}}
+
+Now install and run the translator.
+
+{{< highlight shell >}}
+# Install dependencies
+yum install -q -y rubygems ruby-devel
+
+# Install sensu-translator
+gem install sensu-translator
+
+# Translate the config in /etc/sensu/conf.d to Sensu Go, output to /sensu_config_translated
+sensu-translator -d /etc/sensu/conf.d -o /sensu_config_translated
+{{< /highlight >}}
+
+If translation is successful, you should see a few callouts followed by `DONE!`.
+
+{{< highlight shell >}}
+Sensu 1.x filter translation is not yet supported
+Unable to translate Sensu 1.x filter: only_production {:attributes=>{:check=>{:environment=>"production"}}}
+DONE!
+{{< /highlight >}}
+
+Combine your config into a sensuctl-readable format.
+Note that, for use with `sensuctl create`, Sensu Go resource definitions in JSON format should _not_ have a comma between resource objects.

```

Comment:
Hmm, i would personally prefer to upload into sensu go in pieces, but for the sake of brevity in this guide its probably fine.  Maybe there's a chance to say this is optional, and you also use a piece meal approach to review work flows in a sequential fashion,

Reply:
Added a sentence giving the per-resource-type approach as an option.